### ISSUE TITLE
Double printing progress bar

### ISSUE THREAD

**Issue by kmax12** [FeatureLabs/featuretools#791]
If you force stop a cell in jupyter notebook and then try to rerun, the progress bar get printed on a new line for each update.

<img width="930" alt="Screen Shot 2019-10-29 at 4 37 33 PM" src="https://user-images.githubusercontent.com/371792/67807059-718c2c80-fa6a-11e9-9952-8746b130bd18.png">

It looks like this can be potentially be fixed by adding `tqdm._instance.clear()` after initializing the progress according to this comment: https://github.com/tqdm/tqdm/issues/375#issuecomment-536659188

here is the text code I ran for the screenshot above

```import featuretools as ft
es = ft.demo.load_mock_customer(return_entityset=True, n_transactions=1000000)
feature_matrix, feature_defs = ft.dfs(entityset=es,
                                       target_entity="customers",
                                       verbose=True)```

**Comment by systemshift** [comment]
So I found a python submodule for this to run on native Jupyter https://pypi.org/project/tqdm/#ipython-jupyter-integration
It fixes the issue, and as a side effect the bars are easier on the eyes.

Tests run fine, with the exception of `test_cli.py`. What I would like to know is if there is a use case for tqdm outside of Jupyter. If there is not, then we only need to edit the two failing test units and we are good to go.

**Comment by casperdcl** [comment]
fixed in `tqdm>=4.45.0`, which should not require any special handling on exceptions (no need to catch and clear instances). If it does, please post on https://github.com/tqdm/tqdm/issues/548.

Also just using `from tqdm.auto import tqdm, trange` is probably easiest.
